modules/OCPPConfiguration/main/event_handler.hpp

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+
+#pragma once
+
+#include "generated/types/ocpp.hpp"
+#include "mapping_reader.hpp"
+
+namespace module {
+
+class EventHandler {
+public:
+    EventHandler(const std::filesystem::path& config_mapping_path);
+    void handle_event(const types::ocpp::EventData& event_data, const std::string& user_config_path_string);
```

Comment:
Please add a brief description for public functions

Reply:
Done